src/pages/woong/Detail/Detail_woong.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
```

Comment:
```suggestion
import React, { useState} from 'react';
```
밑에 useState import하는 거 제거하시고 이렇게 써도 됩니다.

Reply:
지금 수정하겠습니다 ㅎㅎㅎ